cli/cmd/stat.go

```diff
@@ -284,6 +320,13 @@ func writeStatsToBuffer(rows []*pb.StatTable_PodGroup_Row, w *tabwriter.Writer,
 				tcpWriteBytes:      getByteRate(r.GetTcpStats().GetWriteBytesTotal(), r.TimeWindow),
 			}
 		}
+		if r.TsStats != nil {
```

Comment:
We still need to modify `key` (line 273) in case of a trafficsplit, because the default key of `namespace` + `name` needs the `leaf` name in order to be unique. But moving the rest of the code down into this block does simplify it, will do 👍